docs/quick-start.md

```diff
@@ -40,7 +40,7 @@ RDDs have _[actions](programming-guide.html#actions)_, which return values, and
 
 {% highlight scala %}
 scala> textFile.count() // Number of items in this RDD
-res0: Long = 126
+res0: Long = 99
```

Comment:
The result should be the same either way. I can't reproduce that and don't see it in the docs either? It's 15 before and you change it to 19.